static/openapi.yaml

```diff
@@ -368,25 +272,10 @@ paths:
                   $ref: '#/components/examples/V1FhirCoverageExample'
           description:  "Coverage FHIR resources bundle"
         '400':
```

Comment:
Above, you have `#/components/schemas/V1FhirCoverage`, but it doesn't map to anything in the schemas.